.buildkite/pipeline.yml

```diff
@@ -115,43 +114,97 @@ steps:
           - maze_output/**/*
 
   - group: ":react: React Native"
+    depends_on: build
     steps:
-      - label: React Native iOS Integration Tests
-        depends_on: build
+      - label: "RN 0.69 :ios: Integration Tests"
+        agents:
+          queue: macos-12-arm
+        env:
+          XCODE_VERSION: 13
+        commands:
+          - chmod +x bin/arm64-macos-bugsnag-cli
+          - bundle install
+          - bundle exec maze-runner features/react-native-ios/rn0_69.feature
+        plugins:
+          artifacts#v1.5.0:
+            download:
+              - bin/arm64-macos-bugsnag-cli
+            upload:
+              - maze_output/**/*
+
+      - label: "RN 0.70 :ios: Integration Tests"
         agents:
           queue: macos-12-arm
         env:
-          XCODE_VERSION: 13.4.1
+          XCODE_VERSION: 13
         commands:
           - chmod +x bin/arm64-macos-bugsnag-cli
           - bundle install
-          - bundle exec maze-runner features/react-native-ios
+          - bundle exec maze-runner features/react-native-ios/rn0_70.feature
         plugins:
           artifacts#v1.5.0:
             download:
               - bin/arm64-macos-bugsnag-cli
             upload:
               - maze_output/**/*
 
-      - label: React Native Android Integration Tests
-        depends_on: build
+      - label: "RN 0.72 :ios: Integration Tests"
         agents:
           queue: macos-12-arm
+        env:
+          XCODE_VERSION: 14
+        commands:
+          - chmod +x bin/arm64-macos-bugsnag-cli
+          - bundle install
+          - bundle exec maze-runner features/react-native-ios/rn0_72.feature
+        plugins:
+          artifacts#v1.5.0:
+            download:
+              - bin/arm64-macos-bugsnag-cli
+            upload:
+              - maze_output/**/*
+
+      - label: "RN 0.69 :android: Integration Tests"
+        commands:
+          - chmod +x bin/arm64-macos-bugsnag-cli
+          - bundle install
+          - bundle exec maze-runner features/react-native-android/rn0_69.feature
+        plugins:
+          artifacts#v1.5.0:
+            download:
+              - bin/arm64-macos-bugsnag-cli
+            upload:
+              - maze_output/**/*
+
+      - label: "RN 0.70 :android: Integration Tests"
+        commands:
+          - chmod +x bin/arm64-macos-bugsnag-cli
+          - bundle install
+          - bundle exec maze-runner features/react-native-android/rn0_70.feature
+        plugins:
+          artifacts#v1.5.0:
+            download:
+              - bin/arm64-macos-bugsnag-cli
+            upload:
+              - maze_output/**/*
```

Comment:
Can I complain about the ordering being 0.72, 0.69 then 0.70?

Reply:
Well they're grouped by platform (ios/android) would we prefer they be numerical order?

Reply:
Oh yes, I see now, I hadn't realised they were different platforms - just ignore the comment.